pandas/_libs/reduction.pyx

```diff
@@ -644,7 +644,7 @@ def compute_reduction(arr, f, axis=0, dummy=None, labels=None):
 
     if labels is not None:
         # Caller is responsible for ensuring we don't have MultiIndex
-        assert not labels._has_complex_internals
+        assert labels.nlevels == 1
```

Comment:
Any reason not to do an isinstance check here? Not a strong blocker usage is mixed; might be nice to settle on one way of doing this

Reply:
because we're in cython and dont want to import non-cython